cliquet/initialization.py

```diff
@@ -367,6 +367,21 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
+def setup_listeners(config):
+    settings = config.get_settings()
+    listeners = [listener.strip() for listener in
+                 settings['event_listeners'].split(',')
+                 if listener.strip()]
+
+    # XXX we should rename it to "RecordChanged" here...
+    from cliquet.events import CollectionChanged
+    #
+    for listener in listeners:
+        listener_mod = config.maybe_dotted(listener)
+        listener = listener_mod.load_from_config(config)
+        config.add_subscriber(listener, CollectionChanged)
```

Comment:
> I really like what you suggested in the design docs about kwargs being passed automatically from config

Not sure what you are talking about. Are you talking about having several sections in the ini ? I am still waiting for feedback on https://github.com/Kinto/kinto/issues/220